tests/testthat/test-dictionary.R

```diff
@@ -233,12 +207,10 @@ test_that("can take the unique loc of 1d arrays (#461)", {
 })
 
 test_that("can use matching functions with strings with different encodings", {
-  utf8 <- "\u00B0C"
-
-  unknown <- utf8
-  Encoding(unknown) <- "unknown"
-
-  latin1 <- iconv(utf8, "UTF-8", "latin1")
+  enc <- encodings()
+  utf8 <- enc$utf8
```

Comment:
I think it'd be a little nicer to just use `enc$utf` in the tests

Reply:
`enc$latin1` -> `enc$latin` too?

Reply:
Yeah, ie don’t assign out the list elements.

Reply:
Ah gotcha, I thought you meant actually change all occurrences of `utf8` to `utf`. I see what you mean now. Done.